homeassistant/components/greeneye_monitor/sensor.py

```diff
@@ -1,7 +1,8 @@
 """Support for the sensors in a GreenEye Monitor."""
 from __future__ import annotations
 
-from typing import Any, Generic, TypeVar
+import typing
+from typing import Any, Generic, Optional, TypeVar
```

Comment:
We could import `cast` from typing so we don't need to import typing on another row.

Reply:
Ok